site/content/entry/yuzu-progress-report-jun-2020/index.md

```diff
@@ -0,0 +1,174 @@
+
+{{< imgs
+    "./res-scaler-removed.png| Until we meet again, Internal Resolution!"
+  >}}
+
+Thanks to users requesting it, [Volca](https://github.com/VolcaEM) added [Open Mods](https://github.com/yuzu-emu/yuzu/pull/4136) [FAQ and Quickstart](https://github.com/yuzu-emu/yuzu/pull/4166) shortcuts on yuzu's interface, a great way to help people just starting to use the emulator, you just need to click on Help!
+
+{{< imgs
+    "./volca-ui.png| Just an easier way. Thanks Volca!"
+  >}}
+  
+On the other hand, [Kewlan](https://github.com/Kewlan) gave us a [mute audio hotkey!](https://github.com/yuzu-emu/yuzu/pull/4164). Additionally, Kewlan also fixed a bug with the [fullscreen hotkey](https://github.com/yuzu-emu/yuzu/pull/4182) where, if you changed it, the old "default" one would still work.
+
+## Dreams and Reality
+
+Project Maribel Hearn is out, and it is in Mainline too! Native 32 bit Switch games start to work, with some of them even being fully playable! This means that games like `New Super Mario Bros. U Deluxe` and `Captain Toad: Treasure Tracker` work almost perfectly and with great performance, while games like `Mario Kart 8 Deluxe` still need more work. 
+
+This has been a very requested feature for a very long time, but implementing it required a major rewrite of the kernel, and official support for ARMv7 was needed on [Dynarmic](https://github.com/MerryMage/dynarmic), our ARM JIT recompiler. This took a tremendous amount of work from several of our developers, including, but not limited to, [Merry](https://github.com/MerryMage), [Lioncache](https://github.com/lioncash), [Blinkhawk](https://github.com/FernandoS27), [bunnei](https://github.com/bunnei), [Rodrigo](https://github.com/ReinUsesLisp), [Tobi](https://github.com/FearlessTobi), [ogniK](https://github.com/ogniK5377), [Morph](https://github.com/Morph1984), and many others. Special mention goes to Merry and Lioncache: they are the soul of this project, and without them we couldn't have even started it. Several of their pull requests are "too technical" to be included in the progress reports, but it should be known that Merry and Lioncache are the main code contributors at the heart of yuzu.
+
+You may not noticed that four months ago, [bunnei](https://github.com/bunnei) and [Tobi](https://github.com/FearlessTobi) laid down the [initial framework](https://github.com/yuzu-emu/yuzu/pull/3478) towards 32 bit support on yuzu by refactoring the ARM interface to support 32 bit and 64 bit JIT backends, implementing a 32 bit ARM interface based on Citra's ARMv6 dynarmic backend by adding CP15 coprocessor support, and implementing several 32 bit SVCs such as `GetThreadPriority32`, `WaitSynchronization32`, and many more.
```

Comment:
This section would be better if broken up into two sentences. Add a period after `yuzu` and then start the next sentence with `They accomplished this by`